homeassistant/components/image/__init__.py

```diff
@@ -0,0 +1,209 @@
+"""The image integration."""
```

Comment:
As a new platform, should it not also be added to `Platform` constant, and to `.core_files.yaml`?

Reply:
It should indeed be added to `.core_files.yaml` 👍 
The PR already adds `image` to the `Platform` enum in `const.py`, do you mean another one?

Reply:
All good. I missed it in `const.py` 👍